server/controllers/admin/users/index.js

```diff
@@ -0,0 +1,269 @@
+/**
+ * @overview Users
```

Comment:
We should decide as a team if we will be going forward with `feature/index.js` as a pattern. If we are it should be formalised and potentially applied to previous modules. 

One disadvantage is fuzzy searches for file names don't work well as everything is called index.js, this isn't a reason to discard the idea though.